src/services/completions.ts

```diff
@@ -1866,6 +1871,7 @@ namespace ts.Completions {
                 return contextualMemberSymbols;
             }
 
+            const membersDeclaredBySpreadAssignment = createMap<boolean>();

```

Comment:
This needs to be Map<true> since we don't use actual value.

Reply:
But it is a Map already, and what do you mean we don't use actual value?

Reply:
I was on mobile and didnt realise its not showing correctly. Sorry about that. It needs to be  `Map<true>`

Reply:
Thanks! Fixed.